src/helpers/transactions.js

```diff
@@ -30,15 +32,7 @@ export function fixInstallments(txns) {
 }
 
 export function sortTransactionsByDate(txns) {
-  const cloned = Array.from(txns);
-  cloned.sort((txn1, txn2) => {
-    if (txn1.date.getTime() === txn2.date.getTime()) {
-      return 0;
-    }
-    return txn1.date < txn2.date ? -1 : 1;
-  });
-
-  return cloned;
+  return orderBy(txns, 'date', 'asc');
```

Comment:
@asfktz nice touch 👍 
I think we should be using `_.orderBy(...)`, as it is simpler.
Also, both functions [seem to expect](https://lodash.com/docs#orderBy) an array for the `iteratees` param, ie, `(txns, ['date'])`.

Reply:
@eshaham thanks (:
I've changed `orderBy ` to `_.orderBy`

The iteratees can also expect a string, this is its signature:
`[iteratees=[_.identity]] (Array[]|Function[]|Object[]|string[])`

Reply:
@asfktz I'm reading my comment above and it doesn't make sense 😆 
I meant to say, use `_.sortBy(...)`, as it is simpler - no need to add `asc` param.

As for iteratees, it shows `string[]`.
Even if it may be working for you without the array, I'm not insisting just for fun.
I don't want it to break when we upgrade to a future version of lodash.